src/python/pants/backend/python/dependency_inference/parse_python_dependencies_test.py

```diff
@@ -329,6 +329,21 @@ def test_imports_from_strings(rule_runner: RuleRunner, min_dots: int) -> None:
             'a.b.c.d.2Bar',
             'a.2b.c.D',
 
+            # Invalid module names that we DO ignore
```

Comment:
Just to make sure we didn't fire a footgin, can you add tests that _asset_ inference still allows trailing dot?

Reply:
There's already a test for that:
https://github.com/pantsbuild/pants/blob/694ea039126ec551601207d89458f0156a839002/src/python/pants/backend/python/dependency_inference/parse_python_dependencies_test.py#L577
https://github.com/pantsbuild/pants/blob/694ea039126ec551601207d89458f0156a839002/src/python/pants/backend/python/dependency_inference/parse_python_dependencies_test.py#L596-L607